lib/mayaUsd/ufe/private/Utils.cpp

```diff
@@ -174,82 +181,82 @@ void applyCommandRestriction(const UsdPrim& prim, const std::string& commandName
+        }
+        catch (const std::exception& e) {
+            std::string err = TfStringPrintf("Failed to rotate prim %s - %s",
+                path.string().c_str(), e.what());
+            UFE_LOG(err.c_str());
+            throw;
+        }
+    }
 }
 
 void scaleOp(const UsdPrim& prim, const Ufe::Path& path, double x, double y, double z)
 {
-	auto primXform = UsdGeomXformCommonAPI(prim);
-	if (!primXform.SetScale(GfVec3f(x, y, z)))
-	{
-		// This could mean that we have an incompatible xformOp in the stack
-		try {
-			primXform = convertToCompatibleCommonAPI(prim);
-			if (!primXform.SetScale(GfVec3f(x, y, z)))
-				throw std::runtime_error("Unable to SetScale after conversion to CommonAPI.");
-		}
-		catch (const std::exception& e) {
-			std::string err = TfStringPrintf("Failed to scale prim %s - %s",
-				path.string().c_str(), e.what());
-			UFE_LOG(err.c_str());
-			throw;
-		}
-	}
+    auto primXform = UsdGeomXformCommonAPI(prim);
+    if (!primXform.SetScale(GfVec3f(x, y, z)))
+    {
+        // This could mean that we have an incompatible xformOp in the stack
+        try {
+            primXform = convertToCompatibleCommonAPI(prim);
+            if (!primXform.SetScale(GfVec3f(x, y, z)))
+                throw std::runtime_error("Unable to SetScale after conversion to CommonAPI.");
+        }
+        catch (const std::exception& e) {
+            std::string err = TfStringPrintf("Failed to scale prim %s - %s",
+                path.string().c_str(), e.what());
+            UFE_LOG(err.c_str());
+            throw;
+        }
+    }
 }
 
 void rotatePivotTranslateOp(const UsdPrim& prim, const Ufe::Path& path, double x, double y, double z)
 {
-	auto primXform = UsdGeomXformCommonAPI(prim);
-	if (!primXform.SetPivot(GfVec3f(x, y, z)))
-	{
-		// This could mean that we have an incompatible xformOp in the stack
-		try {
-			primXform = convertToCompatibleCommonAPI(prim);
-			if (!primXform.SetPivot(GfVec3f(x, y, z)))
-				throw std::runtime_error("Unable to SetPivot after conversion to CommonAPI.");
-		}
-		catch (const std::exception& e) {
-			std::string err = TfStringPrintf("Failed to set pivot for prim %s - %s",
-				path.string().c_str(), e.what());
-			UFE_LOG(err.c_str());
-			throw;
-		}
-	}
+    auto primXform = UsdGeomXformCommonAPI(prim);
+    if (!primXform.SetPivot(GfVec3f(x, y, z)))
+    {
+        // This could mean that we have an incompatible xformOp in the stack
+        try {
+            primXform = convertToCompatibleCommonAPI(prim);
+            if (!primXform.SetPivot(GfVec3f(x, y, z)))
+                throw std::runtime_error("Unable to SetPivot after conversion to CommonAPI.");
+        }
+        catch (const std::exception& e) {
+            std::string err = TfStringPrintf("Failed to set pivot for prim %s - %s",
+                path.string().c_str(), e.what());
+            UFE_LOG(err.c_str());
+            throw;
+        }
+    }
```

Comment:
Please ignore the tab conversion to dot changes by enabling "Hide whitespace changes" on github.